tutorials/jitsi-ubuntu-jammy-jellyfish/index.mdx

```diff
@@ -0,0 +1,89 @@
+The tool provides features like:
+
+- screen sharing
+- presentations
+- inviting users to a conference through a custom URL
+- editing documents together using Etherpad
+- integrated chat
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [configured your SSH key](/console/my-project/how-to/create-ssh-key/)
+  - You have a [Scaleway Instance](/compute/instances/how-to/create-an-instance/) running Ubuntu Jammy Jellyfish (for better performance, we recommend that you use an instance with **at least 4 GB of RAM** - DEV1-M or larger)
+  - You have a domain or subdomain pointed to your Instance
+</Message>
+
+<Message type="note">
+  This tutorial shows the installation of Jitsi Meet on Ubuntu Jammy Jellyfish. If you prefer the installation of Jitsi Meet on Ubuntu Bionic Beaver, follow [this tutorial](/tutorials/jitsi-ubuntu-bionic).
+</Message>
+
+### Installing Jitsi Meet
+
+1. Configure the hostname of the server corresponding to your domain or subdomain.
+    ```
+    hostnamectl set-hostname jitsi
+    sed -i 's/^127.0.1.1.*$/127.0.1.1 jitsi.mydomain.tld jitsi/g' /etc/hosts
+    sed -i 's/^127.0.0.1.*$/127.0.0.1 localhost jitsi.mydomain.tld jitsi/g' /etc/hosts
+    ```
+2. Update the apt packet cache and upgrade the software installed on the instance to the latest version available on the Debian repositories.
+    ```
+    apt update && apt upgrade
+    ```
+3. Install an [Nginx](https://nginx.org) web server. The Jitsi installation tool handles the Nginx configuration if it is present on the system.
+    ```
+    apt install -y nginx
+    systemctl start nginx.service
+    systemctl enable nginx.service
+    ```
+
+    <Message type="important">
+        If Nginx or Apache are not installed, Jitsi Meet will automatically install [Jetty](https://www.eclipse.org/jetty/) during the installation.
+    </Message>
+
+4. Download and install the Prosody package repository. 
+    ```
+    echo deb http://packages.prosody.im/debian $(lsb_release -sc) main | sudo tee -a /etc/apt/sources.list
+    wget https://prosody.im/files/prosody-debian-packages.key -O- | sudo apt-key add -
+    apt install lua5.2
+    ```
+5. Download the APT key and set up the repositories of Jitsi:
+    ```
+    wget -qO - https://download.jitsi.org/jitsi-key.gpg.key | apt-key add -
+    sh -c "echo 'deb https://download.jitsi.org stable/' > /etc/apt/sources.list.d/jitsi-stable.list"
+    apt update
+    ```
+6. Launch the installation of Jitsi Meet.
+    ```
+    apt install -y jitsi-meet
+    ```
+
+    When asked, enter:
+
+    - The FQDN of your instance. For example `jitsi.mydomain.tld` and press Enter. 
+    - When asked about the SSL certificate choose the `Let's Encrypt certificates` option and press Enter.
+    - Your email address.
+    - Your phone number (optional).
```

Comment:
```suggestion
    - enter your phone number (optional).
```

Reply:
I didn't put the verb enter because the list starts with "when asked, enter : "